### DISCUSSION TITLE
How do I read the confusing time axis on Dashboard task stream

### DISCUSSION THREAD

**Discussion by ketozhang** [dask/dask#9875] [Q&A]
I'm getting a pretty non-intuitive tick labels for the time axis on the task stream figure:

![image](https://user-images.githubusercontent.com/13506487/214712394-0f63c85d-e9cb-48e3-9f18-97deccf51057.png)

* What I figure out so far is we have a seconds formatted as `:mm`
* What's up with `1/01`? This looks like the origin (0 seconds).
* Why do is there `:58` before this.

Also, the [example in docs](https://distributed.dask.org/en/stable/diagnosing-performance.html) better formatting, but the 0ms mark restarts every 1 second?!
![image](https://user-images.githubusercontent.com/13506487/214713076-14074f5d-3a9f-4905-b57e-14a13e977f93.png)

**Comment by ncclementi** [comment]
hi @ketozhang  , Dask has a [Discourse forum](https://dask.discourse.group/t/introduce-yourself/22/23) for usage-questions.

We'll see you there! :)
